### DISCUSSION TITLE
Nuxt3 integrated typescript leads to TS1109: Expression expected. and other problems

### DISCUSSION THREAD

**Discussion by xhc-code** [nuxt/nuxt#20774] [Questions]
environment:
nuxt3 + vue3 + vue-tsc1.6.4 + typescript5.0.4

![image](https://github.com/nuxt/nuxt/assets/28866025/0e901a44-117c-44b7-b79a-149cba15d8f0)

A simple import can go wrong, which makes me at a loss. This problem can be solved by downgrading typescript5.0.4 to version 4.x (currently downgrading typescript to 4.9.5).
Does anyone know if this is a problem caused by vue-tsc or nuxt or typescript? Have you met any big guys?

**Comment by storageddd** [comment]
I have same issue. Solve it please
